src/modules/documentExecution/dashboard/DashboardHelpers.ts

```diff
@@ -142,4 +142,19 @@ export const getFormattedOutputParameters = (documentOutputParameters: IDashboar
             type: documentOutputParameter.type
         }
     })
+}
+
+export const apllyDashboardViewToModel = (dashboardModel: IDashboard, view: IDashboardView | null) => {
```

Comment:
Could you please fix the typo in the name? Thank you

Reply:
Fixed